src/opendap/logging/ServletLogUtil.java

```diff
@@ -568,9 +572,25 @@ public static void useCombinedLog(boolean value) {
         log.info("Combined OLFS/BES Log Is {}", value ? "ENABLED." : "DISABLED");
     }
 
-    public static void useDualCloudWatchLogs(boolean value) {
+    public static void setUseDualCloudWatchLogs(boolean value) {
```

Comment:
I know that this is totally a style thing 
```
SomeType ###
void set###(SomeType st);
SomeType get###(); 
```
But some part of me always wants to:
```
SomeType ###
void ###(SomeType st);
SomeType ###(); 
```
I suspect that, in general, that's consider bad...